dev_docs/cxg.md

```diff
@@ -119,46 +117,61 @@ This optional field contains a copy of the category color table, which MAY be us
 
-### Corpora metadata property
 
-A CXG containing a Corpora dataset will contain a property in the __cxg_group_metadata__ field named `corpora`. The value will be a JSON encoded string, which in turn contains all properties defined in the [Corpora AnnData uns](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md#uns) container. For example:
+These Corpora metadata properties WILL exist in the CXG `__cxg_group_metadata__` field named `corpora`: 
+* `schema_version`
+* `title`
+* `X_normalization`
+* `X_approximate_distribution`
 
+These Corpora metadata properties MAY exist in the CXG `__cxg_group_metadata__` field named `corpora`:
+* `contributors`
+* `default_embedding`
+* `default_field`
+* `project_name`
+* `project_description`
+* `project_links`
+* `preprint_doi`
+* `publication_doi`
+* `tags`
+
+For example:
 ```
 {
     "corpora": {
-        "version": {
-            "corpora_schema_version": "1.0.0",
-            "corpora_encoding_version": "0.1.0",
-        }
+        "schema_version": "2.0.0",
+        "title": "Analysis of 8 datasets of healthy and diseased mouse heart",
+        "X_normalization": "CPM",
+        "X_approximate_distribution": "normal",
+        "project_name": "Diseased Mouse Heart Study"
     }
 }
 ```
 
-The `corpora` metadata, if present, MUST contain the version information. Optionality of other values in this object will follow the specifications set forth in the relevant Corpora schema specification (ie, optional fields are optional, required are present, etc), with the following changes:
-* the contents of `corpora_encoding_version` MUST be identical to the `cxg_version`, as this field is defined as the current object encoding version, *NOT* the source data encoding version.
-* the entire encoding will be JSON, rather than a hybrid Python/JSON encoding, but will otherwise follow the data structure defined by the AnnData Corpora encoding.
-* the `<obs_column>_colors` will be omitted in favor of `cxg_category_colors`
-
 ### Other Corpora fields
 
-All other Corpora schema fields will be encoded into a CXG using the conventions defined in the [Corpora Schema AnnData Implementation](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md). For example, fields in `AnnData.obs` will be encoded in the CXG `obs`array as defined [above](#obs-and-var).
+All other Corpora schema fields will be encoded into a CXG using the conventions defined in the [Corpora Schema](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/2.0.0/corpora_schema.md). For example, fields in `AnnData.obs` will be encoded in the CXG `obs`array as defined [above](#obs-and-var).
 
 ### Compatibility with CXG 0.1.0
```

Comment:
In fact, since CXG is a private format, do we need to continue to support CXG 0.1.0 at all? If all CXG files are being updated to CXG 0.2.0, can we eliminate this section altogether?

Reply:
Per Slack discussion cxg 0.1.0 files will persist for a while, so keeping this in here for now.